CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # node-pre-gyp changelog
 
-## Unreleased - 2.0.0
+## Unreleased
+
```

Comment:
I think we can remove this because it will no longer be unreleased after merging this PR
```suggestion
```